docs/en/observability/view-monitor-status.asciidoc

```diff
@@ -43,36 +43,47 @@ The information displayed includes the recent status of a host or endpoint, when
 was last checked, its URL, and, if applicable, the TLS certificate expiration time. There is
 also a sparkline showing downtime history.
 
+[TIP]
+=====
+Select a tag in the monitor list to display only those monitors the tag is assigned to.
+This can be valuable if you have assigned tags based on which cloud provider the monitor
+is hosted on.
```

Comment:
What about rewording this TIP to something like this?

> Use monitor tags to display a custom assortment of monitors; for example, consider assigning tags based on a monitor's hosted cloud provider – making it easy to quickly see all monitors hosted on GCP, AWS, etc.

Reply:
Great suggestion. Thanks.